src/integTest/groovy/org/gradle/api/plugins/clover/CloverPluginIntegSpec.groovy

```diff
@@ -284,4 +328,37 @@ class CloverPluginIntegSpec extends Specification {
             conn.close()
         }
     }
+    
+    /**
+     * @returns true if the zip file contains a file with the given name.
+     */
+    private boolean zipContains(File zip, String fileName) {
+        def zipFile = new ZipFile(zip)
+        ZipEntry entry = zipFile.entries().find { ZipEntry zipEntry ->
+            zipEntry.name == fileName
+        }
+        return entry != null
+    }
+    
+    // http://www.java-tips.org/java-se-tips/java.util.zip/how-to-extract-file-files-from-a-zip-file-3.html
```

Comment:
This code snippet has unknown copyright license. I suspect that it's copyrighted as there's a "(c) 2015 Java Tips" in the footer. I would recommend rewriting this code.